WordPress/Classes/Extensions/NSAttributedString+StyledHTML.swift

```diff
@@ -19,22 +19,23 @@ extension NSAttributedString {
     class func attributedStringWithHTML(_ htmlString: String, attributes: StyledHTMLAttributes?) -> NSAttributedString {
         let styles = styleTagTextForAttributes(attributes)
         let styledString = styles + htmlString
-        let attributedString = try! NSMutableAttributedString(
+        guard let attributedString = try? NSMutableAttributedString(
             data: styledString.data(using: String.Encoding.utf8)!,
```

Comment:
Perhaps we should `if let` this data too, to avoid another force unwrap – just to be safe?